dist/src/main/dist/conf/simulator.properties

```diff
@@ -171,7 +171,7 @@ MACHINE_SPEC=hardwareId=c3.xlarge,imageId=us-east-1/ami-1b3b2472
 #                                   git=myRepository/myBranch - to build a version from a branch in a specific repository.
 #   Use GIT_CUSTOM_REPOSITORIES to specify custom repositories. The main Hazelcast repository is always named "origin".
 #
-HAZELCAST_VERSION_SPEC=maven=3.4.2
+HAZELCAST_VERSION_SPEC=maven=3.5-SNAPSHOT
```

Comment:
I'm quite confused now. If Jenkins has its own properties, why does `outofthebox` hurt then? How I understood this setting is just effective on manual runs if you don't provide your own `simulator.properties` or use the git command line argument.

So what should we put in here? I have in mind that we said latest stable release, so users will not have to deal with unstable HZ versions. So it would still be `3.4.2` for now.